first_project.html

```diff
@@ -3,7 +3,7 @@
 title: Osoc 2019 - Configurator for individual route planners
 ---
 
-<section id="third_project">
+<section class = "projectsPages" id="first_project">
```

Comment:
Be careful about spacing between the attribute and its value: there shouldn't be a space :)